Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli

```diff
@@ -79,15 +83,15 @@ class Surface
 void Surface::ApplySpecularAA()
 {
     // Constants for formula below
-    const float screenVariance = 0.25f;
-    const float varianceThresh = 0.18f;
+    const real screenVariance = 0.25;
+    const real varianceThresh = 0.18;
 
     // Specular Anti-Aliasing
-    float3 dndu = ddx_fine( normal );
-    float3 dndv = ddy_fine( normal );
-    float variance = screenVariance * (dot( dndu , dndu ) + dot( dndv , dndv ));
-    float kernelRoughnessA2 = min(2.0 * variance , varianceThresh );
-    float filteredRoughnessA2 = saturate ( roughnessA2 + kernelRoughnessA2 );
+    real3 dndu = real3(ddx_fine( float3(normal) ));
```

Comment:
Note that down/upcasting to and from `half` precision floats will reduce perf. In this case you might be better off leaving `normal` as a `float3`.